### DISCUSSION TITLE
user_id and CSV format

### DISCUSSION THREAD

**Discussion by bardware** [kheina-com/Blue-Blocker#243] [Q&A]
I believe, in the past, I imported a safe list in Firefox browser that looked like
```
user_id,screen_name
,POTUS
```

Today, I tried to import a safe list in Edge browser but this did not work. How does the CSV file has to look, that it can be imported? Do I have to provide the ID or does the handle suffice?

**Comment by rougetimelord** [comment]
Currently the import feature requires a user id on each line. It might be possible to enhance the feature to work with either handles or user ids in the future though
